src/routes/(authentication)/login/loading/+page.svelte

```diff
@@ -31,6 +32,15 @@
     }
     console.log('hashParts', hashParts);
     failed = await providerInstance.authenticate(hashParts);
+
+    let filtersJson = localStorage.getItem('filters');
+    console.log('filtersJson: ' + filtersJson);
```

Comment:
If you want to keep the logging can you change it to debug?

Reply:
I'll get rid of this